### PR TITLE
Fixed issue#8

### PR DESCRIPTION
**Pull request by Dipti821**

### CODE REVIEW

File: DSA-Code-Snippet/C++/concatenation.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+## Question
+
+Given an integer array nums of length n, you want to create an array ans of length 2n where ans[i] == nums[i] and ans[i + n] == nums[i] for 0 <= i < n (0-indexed).
+
+Specifically, ans is the concatenation of two nums arrays.
+
+Return the array ans.
+
```
Comment (Manikant25) [inline DSA-Code-Snippet/C++/concatenation.md:8]: Remove extra spaces

**Comment by Manikant25** [comment]
Review [contributing guildlines](https://github.com/DSCSRMNCR/DSA-Code-Snippet/blob/main/CONTRIBUTING.md)

File: DSA-Code-Snippet/C++/concatenation.md (hunk 1 above)
Comment (Dipti821) [reply DSA-Code-Snippet/C++/concatenation.md:8]: ohk..

Comment (Dipti821) [reply DSA-Code-Snippet/C++/concatenation.md:6]: Done!!

**Comment by Manikant25** [comment]
Looks good!

**Pull request merged**